src/ctia/import/threatgrid/feed/sightings.clj

```diff
@@ -7,76 +7,53 @@
             [cheshire.core :as json]
             [clojure.edn :as edn]))
 
-(defn make-feed-sighting
-  [description feed-uri timestamp observable indicator]
-  (let [sighting {:description description
-                  :type "sighting"
-                  :source "ThreatGrid Feed"
-                  :source_uri feed-uri
-                  :tlp "green"
-                  :confidence "High"
-                  :timestamp timestamp
-                  :observables [observable]
-                  :indicators [indicator]}]))
+(defn entry->relations
+  [feed entry source-observable relation-type related-type related-key]
+  (mapv (fn [related-value]
+          {:origin (:source feed)
+           :origin_uri (:source_uri feed)
+           :source source-observable
+           :relation relation-type
+           :related {:value related-value
+                     :type related-type}})
+        (related-key entry)))
 
 (defn transform
-  [entries
-   observable-type
-   observable-value
-   indicator-id
-   indicator-source
-   source-uri & {:keys [tlp confidence]
-                 :or {tlp "green"
-                      confidence "High"}}]
-  (map
-   (fn [entry]
-     (let [{:keys [description timestamp]} entry]
-       {:description description
-        :type "sighting"
-        :source "ThreatGrid"
-        :source_uri source-uri
-        :tlp tlp
-        :confidence confidence
-        :timestamp timestamp
-        :observables [{:type observable-type
-                       :value (observable-value entry)}]
-        :indicators [{:confidence "High"
-                      :source indicator-source
-                      :indicator_id indicator-id}]}))
-   entries))
-
-(defn entries->sightings
-  [entries observable-type observable-field
-   indicator-id indicator-source source-uri & {:as options}]
-  (apply transform entries observable-type observable-field
-         indicator-id indicator-source source-uri options))
+  [feed & {:keys [tlp confidence]
+           :or {tlp "green"
+                confidence "High"}}]
+  (let [{:keys [title description source source_uri
```

Comment:
This is exactly what I needed--I had tried destructuring it initially, but all the tutorials and notes that I could find on it were for let blocks, and I ended up having an uneven number of symbols in the form.   In particular, in the example  `{foo :foo bar :bar :as baz} :baz`, the trailing `:baz` is what I was missing--this is a very helpful comment, thank you.